### PR TITLE
WRO-13694: 3D Gallery: Create separate sample.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
Enact-DCO-1.0-Signed-off-by: Alexandru Morariu alexandru.morariu@lgepartner.com

### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
I moved the Gallery3D app from sandstone repo to samples.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-13694

### CODE REVIEW

File: sandstone/gallery-3d/resources/ilibmanifest.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"files": []
+}
```
Comment (adrian-cocoara-lgp) [inline sandstone/gallery-3d/resources/ilibmanifest.json:3]: add an empty space

File: sandstone/gallery-3d/src/App/package.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+	"main": "App.js"
+}
```
Comment (adrian-cocoara-lgp) [inline sandstone/gallery-3d/src/App/package.json:3]: add an empty space

**Comment by adrian-cocoara-lgp** [comment]
We also need a fallback for images when image fetching fails

File: sandstone/gallery-3d/resources/ilibmanifest.json (hunk 1 above)
Comment (alexandrumorariu) [reply sandstone/gallery-3d/resources/ilibmanifest.json:3]: Done

File: sandstone/gallery-3d/src/App/package.json (hunk 2 above)
Comment (alexandrumorariu) [reply sandstone/gallery-3d/src/App/package.json:3]: Done

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**